lib/Doctrine/DBAL/Platforms/SQLServerPlatform.php

```diff
@@ -230,12 +230,11 @@ public function getDropIndexSQL($index, $table = null)
 
         return sprintf(
             <<<SQL
-IF EXISTS (SELECT * FROM sysobjects WHERE name = '%s')
-    ALTER TABLE %s DROP CONSTRAINT %s
-ELSE
-    DROP INDEX %s ON %s
-SQL
-            ,
+            IF EXISTS (SELECT * FROM sys.sysobjects WHERE name = '%s')
```

Comment:
normalize all system where to use `sys.` prefix

Reply:
Ok, why not, but what does that change? If you confirm to me that it is not directly related to the fix, I will move it to a separate commit.